app/services/messaging/twilio/api.rb

```diff
@@ -2,13 +2,12 @@ class Messaging::Twilio::Api < Messaging::Channel
   include Rails.application.routes.url_helpers
   include Memery
 
-  def send_message(recipient_number:, message:)
+  def send_message(recipient_number:, message:, &with_communication_do)
```

Comment:
Consider `after_send` instead of `with_communication_do`. I know that means you have to "know" that the `communication` is exposed in the block, but I think the more conventional name is worth it.

Reply:
Consider `after_send` instead of `with_communication_do`. I know that means you have to "know" that the `communication` is exposed in the block, but I think the more conventional name is worth it.

Reply:
In fact, it doesn't change the caller interface at all, so I'd strongly recommend it 😅

Reply:
In fact, it doesn't change the caller interface at all, so I'd strongly recommend it 😅

Reply:
The `after_send` convention coming from rails callbacks? I ended up on the `_do` since it hints at a ruby block that does something to a `communication` but `after_send` also works if it makes sense in rails' context.